image.go

```diff
@@ -12,33 +13,49 @@ import (
 	"os"
 
 	"github.com/disintegration/gift"
+	"golang.org/x/image/bmp"
 )
 
-func resizeAndRotate(img image.Image, width, height int) image.Image {
-	g := gift.New(
-		gift.Resize(width, height, gift.LanczosResampling),
-		//gift.UnsharpMask(1, 1, 0),
-		gift.Rotate180(),
-	)
+func resizeAndRotate(img image.Image, width, height int, devname string) image.Image {
+	g, _ := deviceSpecifics(devname, width, height)
 	res := image.NewRGBA(g.Bounds(img.Bounds()))
 	g.Draw(res, img)
 	return res
 }
 
+func deviceSpecifics(devName string, width, height int) (*gift.GIFT, error) {
```

Comment:
This function reads in the Device name and returns a new `gift.GIFT` object (I'm bad with Golang terms) based on device.